composer.json

```
@@ -10,7 +10,8 @@
   "prefer-stable": true,
   "minimum-stability": "dev",
   "require": {
-    "pimcore/pimcore": "^11.0"
+    "pimcore/pimcore": "^11.0",
+    "symfony/dotenv": "^6.1"
```

I think we should use `v6.2`, as `v6.1` [is not maintained anymore](https://symfony.com/releases/6.1).

```suggestion
    "symfony/dotenv": "^6.2"
```